package.json

```diff
@@ -12,12 +12,15 @@
     "src"
   ],
   "scripts": {
+    "clean": "rimraf dist",
     "start": "tsdx watch --verbose --noClean --format umd --name JBrowsePluginMyProject --onFirstSuccess \"yarn serve --cors --listen 9000 .\"",
+    "prebuild": "npm run clean",
     "build": "tsdx build --format cjs,esm,umd --name JBrowsePluginMyProject",
     "test": "tsdx test",
     "lint": "tsdx lint",
     "prepublishOnly": "yarn test",
-    "postinstall": "jbrowse-plugin-postinstall"
+    "postinstall": "jbrowse-plugin-postinstall",
+    "postversion": "git run build && git push --follow-tags"
```

Comment:
What if we have `npm run build` in a "prepare" script instead? Seems to be the preferred method here: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#devdependencies

Reply:
this is fine with me, i was just surprised when i had it in preversion (which is what many of our data parser modules have) that it was bundling the wrong version so i put it in postversion here. will move to prepare